### PR TITLE
partial outline of conformance

### PR DESCRIPTION
**Pull request by isaacvetter**
note that privacy.html currently links to must-support anchor.

How much content do we need for must-support?

### CODE REVIEW

File: input/pagecontent/conformance.md [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+
+
+See: [the RESTful API](http://hl7.org/fhir/R4/http.html)
+
+
+## Conformance Artifacts
+The [Artifacts](artifacts.html) page lists the IPA Profiles that have been defined for this implementation guide. Core Profile [StructureDefinitions](http://hl7.org/fhir/R4/structuredefinition.html) defines the minimum elements, extensions, vocabularies and value sets which SHALL be present when using the profile. Many Profile pages also contain additional guidance.
+
+The Profile elements consist of both Mandatory and Must Support elements. Mandatory elements are elements with an minimum cardinality of 1 (min=1). The base [FHIR Must Support](http://hl7.org/fhir/R4/profiling.html#mustsupport) guidance requires specifications to define exactly the support expected for profile elements labeled Must Support. The sections below illustrate how these elements are displayed and define the rules for interpreting profile elements and subelements labeled Mandatory and Must Support for requesters and responders.
```
Comment (mrinnetmaki) [inline input/pagecontent/conformance.md:15]: *a* minimum cardinality

File: input/resources/CapabilityStatement-ipa-server.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
...
+      "documentation" : "The IPA Server **SHALL**:\n\n1. Support the IPA Patient resource profile.\n1. Support at least one additional resource profile from the list of IPA Profiles.\n1. Implement the RESTful behavior according to the FHIR specification.\n1. Return the following response classes:\n   - (Status 400): invalid parameter\n   - (Status 401/4xx): unauthorized request\n   - (Status 403): insufficient scope\n   - (Status 404): unknown resource\n   - (Status 410): deleted resource.\n1. Support json source formats for all US Core interactions.\n1. Declare a CapabilityStatement identifying the list of profiles, operations, search parameter supported.\n\nThe IPA Server **SHOULD**:\n\n1. Support xml source formats for all IPA interactions.\n1. Identify the IPA profiles supported as part of the FHIR `meta.profile` attribute for each instance.\n1. Support xml resource formats for  questionnaire interactions.",
```
Comment (mrinnetmaki) [inline input/resources/CapabilityStatement-ipa-server.json:46]: 👍

File: input/pagecontent/conformance.md [hunk 3]
```diff
@@ -1,2 +1,45 @@
...
+
+The [Capability Statement page](CapabilityStatement-ipa-server.html) outlines conformance requirements and expectations for the IPA Servers. The [IPA Server CapabilityStatement](CapabilityStatement-ipa-server.html) identifies the specific profiles and RESTful transactions that need to be supported. Note that country-specific profiles must define terminology bindings and invariants.
```
Comment (mrinnetmaki) [inline input/pagecontent/conformance.md:17]: The links seem identical? Just one would be enough?

File: input/pagecontent/conformance.md [hunk 4]
```diff
@@ -1,2 +1,45 @@
...
+
+The [Capability Statement page](CapabilityStatement-ipa-server.html) outlines conformance requirements and expectations for the IPA Servers. The [IPA Server CapabilityStatement](CapabilityStatement-ipa-server.html) identifies the specific profiles and RESTful transactions that need to be supported. Note that country-specific profiles must define terminology bindings and invariants. 
+
+## Profile Support + Interaction Support
+Systems deploy, and support, the IPA Profiles to represent clinical information and the IPA RESTful interactions to access the information. Systems that implement both can claim conformance to IPA. This is done by implementing all of the IPA CapabilityStatement into their capabilities.
+
+To claim conformance to a US Core Profile a server:
```
Comment (mrinnetmaki) [inline input/pagecontent/conformance.md:22]: To claim conformance to *an IPA Profile* a server

File: input/pagecontent/conformance.md [hunk 5]
```diff
@@ -1,2 +1,45 @@
...
+
+* SHALL Be able to populate all profile data elements that are mandatory and/or flagged as Must Support as defined by that profile’s StructureDefinition.
```
Comment (mrinnetmaki) [inline input/pagecontent/conformance.md:24]: SHALL *be* able

File: input/pagecontent/conformance.md [hunk 6]
```diff
@@ -1,2 +1,45 @@
...
+* SHALL declare conformance with the the US Core Server Capability Statement by including its official URL in the server’s CapabilityStatement.instantiates element: <todo: need a uri, here, US Core's is: http://hl7.org/fhir/uv/core/CapabilityStatement/us-core-server>
```
Comment (mrinnetmaki) [inline input/pagecontent/conformance.md:25]: 👍 Can we just refer to a page in this IG?

File: input/pagecontent/conformance.md [hunk 7]
```diff
@@ -1,2 +1,45 @@
...
+    * The IPA Profile’s official or “canonical” URL can be found on each IPA Profile page
+
+<todo: example CapabilityStatement snippet for a server conforming to the IPA Patient Profile>
+
+
+## Must-support
+In the context of IPA, Supported on any data element SHALL be interpreted as follows:
+* Responders SHALL be capable of including the data element as part of the query results as specified by the IPA conformance resources.
+* Requestors SHALL be capable of processing resource instances containing the data elements. In other words Requestors SHOULD be capable of displaying the data elements for human use or storing it for other purposes.
+* In situations where information on a particular data element is not present and the reason for absence is unknown, Responders SHALL NOT include the data elements in the resource instance returned as part of the query results.
```
Comment (mrinnetmaki) [inline input/pagecontent/conformance.md:38]: It's not immediately clear to me what the *information on a particular data element* refers to.
What is this sentence for?

File: input/pagecontent/conformance.md [hunk 8]
```diff
@@ -1,2 +1,45 @@
...
+* When querying Responders, Requestors SHALL interpret missing data elements within resource instances as data not present in the Responder’s systems.
+* In situations where information on a particular data element is missing and the Responder knows the precise reason for the absence of data, Responders MAY send the reason for the missing information using values from the value set where they exist or using the dataAbsentReason extension.
+* Requestors SHALL be able to process resource instances containing data elements asserting missing information.
+
+NOTE: Responders who do not have the capability to store or return a data element tagged as Supported in IPA profiles can still claim conformance to the IPA profiles per the IPA  conformance resources.
```
Comment (mrinnetmaki) [inline input/pagecontent/conformance.md:43]: 👍

**Review by mrinnetmaki — COMMENTED** [review]
Good stuff! Not marking aas approved, as there are still a couple of todos.

File: input/pagecontent/conformance.md (hunk 7 above)
Comment (isaacvetter) [reply input/pagecontent/conformance.md:38]: Hey @mrinnetmaki, I think that "information on a particular data element" refers to metadata about a data element that could explain why it's not present.

File: input/pagecontent/conformance.md [hunk 9]
```diff
@@ -12,17 +12,17 @@ See: [the RESTful API](http://hl7.org/fhir/R4/http.html)
...
 
-The [Capability Statement page](CapabilityStatement-ipa-server.html) outlines conformance requirements and expectations for the IPA Servers. The [IPA Server CapabilityStatement](CapabilityStatement-ipa-server.html) identifies the specific profiles and RESTful transactions that need to be supported. Note that country-specific profiles must define terminology bindings and invariants. 
+The [Capability Statement page](CapabilityStatement-ipa-server.html) outlines conformance requirements and expectations for the IPA Serversas well as the specific profiles and RESTful transactions that need to be supported. Note that country-specific profiles must define terminology bindings and invariants.
```
Comment (mrinnetmaki) [inline input/pagecontent/conformance.md:17]: IPA `Servers as` well

**Review by mrinnetmaki — APPROVED** [review]
Still one minor typo. :)

File: input/pagecontent/conformance.md (hunk 7 above)
Comment (mrinnetmaki) [reply input/pagecontent/conformance.md:38]: After some digging I found https://www.hl7.org/fhir/us/core/general-guidance.html#missing-data which describes the use case more clearly and in more detail (to me, at least).

Comment (mrinnetmaki) [reply input/pagecontent/conformance.md:38]: See also https://chat.fhir.org/#narrow/stream/179175-argonaut/topic/Data.20Absent.20Reason.20-.20why.3F/near/248670602 for some context.

**Pull request merged**